test/integration/onchain/demandA.test.ts

```diff
@@ -8,22 +8,29 @@ import { seed, cleanup, parametersAttachmentId, seededDemandAId } from '../../se
 
 import { selfAddress, withIdentitySelfMock } from '../../helper/mock.js'
 import Database, { DemandRow } from '../../../src/lib/db/index.js'
-import ChainNode from '../../../src/lib/chainNode.js'
 import { pollTransactionState, pollDemandState, pollDemandCommentState } from '../../helper/poll.js'
 import { withAppAndIndexer } from '../../helper/chainTest.js'
 import { container } from 'tsyringe'
 import { filterRejectedAndAcceptedPromises } from '../../helper/parallelTests.js'
+import { withProxy } from '../../helper/proxy.js'
+import ExtendedChainNode from '../../helper/testInstanceChainNode.js'
+import env from '../../../src/env.js'
+import { logger } from '../../../src/lib/logger.js'
 
-describe('on-chain', function () {
+describe('on-chain via proxy', function () {
```

Comment:
let's reduce the churn for when the non-proxy version goes away. Can we call these `on-chain` and then the new test suite `on-chain proxyless` or similar please